qa/automation/CareerPortal.Tests.Scenarios/Burlaka.feature

```diff
@@ -0,0 +1,119 @@
+Then following "<logo-containing element>" with "<logo>" logo should be displayed in the footer:
+|logo-containing element|logo                  | 
+|link button            |"Planet"              | 
+|link button            |hh                    | 
+|link button            |f                     | 
+|link button            |"Camera"              |
+|link button            |vk                    |
+|link button            |in                    |
+|scroll-to-top button   |Quantori              |
+|mailto button          |"Envelope"            | 
+
+Scenario:1.2.3_03 Footer link "Data Privacy policy" provides access to a readable and downloadable document
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+When user clicks a link with "Data Privacy policy" text in the footer
+Then a pdf file "Quantori Privacy Policy" should be displayed in the new window
+And user should be able to download a "Quantori Privacy Policy" pdf file
+
+Scenario:1.2.3_04 Footer link "Cookies Settings" opens a pop-up window with cookie settings
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+When user clicks a link with "Cookies Settings" text in the footer
+Then a pop-up window with the following text should be displayed in the same window:
+"""
+We use cookies to enhance the functionality of the website. 
+Some cookies are necessary for our site to work while others are optional to aid in the user experience
+For overall information about how we protect your data please review our Data Privacy Policy.
+"""
+
+Scenario:1.2.3_05 Footer link "Cookies Settings" allows a user to submit selected by default (necessary) cookies preferences
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+And user clicked a link with "Cookies Settings" text in the footer
+When a pop-up window with selected by default "Necessary cookies" checkbox and "Other cookies" checkbox is displayed
+Then user should click "Submit preferences" button
+
+Scenario:1.2.3_06 Footer link "Cookies Settings" allows a user to choose and submit other cookies preferences
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+And user clicked a link with "Cookies Settings" text in the footer
+When a pop-up window with selected by default "Necessary cookies" checkbox and "Other cookies" checkbox is displayed
+Then user should select "Other cookies" checkbox 
+And user should click "Submit preferences" button
+
+Scenario:1.2.3_07 Pop-up window "Cookies Settings" contains a link to "Data Privacy policy" pdf file
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+And user clicked a link with "Cookies Settings" text in the footer
+When a pop-up window with cookies settings is displayed
```

Comment:
"When steps are used to describe an event or an action. This can be a person interacting with the system, or it can be an event triggered by another system."

Event or action - where event or action in your step? 

This can be a person interacting with the system, or it can be an event triggered by another system." 

No conditions for your decision.

Reply:
Also, your test checks nothing. 
You should check how works link "Data Privacy policy"  - the user should click on it and see results.
All visibility elements of the window should be covered in case 04.